BraintreeDropIn.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "BraintreeDropIn"
-  s.version          = "9.12.2"
+  s.version          = "9.13.0"
```

Comment:
We don't want to manually change the version, this is updated automatically via CI when we do a release